### ISSUE TITLE
Word file having some problem while using addimage (Docx)

### ISSUE THREAD

**Issue by shihabudheenf** [PHPOffice/PHPWord#1389]
This is:

I am creating dynamic word using phpWord and it is working fine for text, when I am using `$header->addImage` and then word is creating and image is displays inside word properly.

But the problem is,  when I am downloading the same file, it shows as zip extension. (in browser)

When I am using google preview option to preview document, it is not working and showing as not supported format.

When opening created document from ms word, it is showing without problem. 

I want to preview my document in my web application and currently not working when using image inside doc. 

Please help me to resolve the issue. 

`$phpWord = new PhpWord();`
 `$section = $phpWord->addSection();`
` $header = $section->addHeader();`
` $header->addImage("app/assets/img/" . $settings->header_image, array('width' => 530, 'height' => 75, 'marginTop' => -1
            , 'marginLeft' => -1, 'marginRight' => -1));`
`$objWriter = IOFactory::createWriter($phpWord, 'Word2007');`
        `$path = storage_path("app/documents/$document->id.docx");`
        `$objWriter->save($path);`

* PHP version: 7
* PHPWord version: 0.14

**Comment by troosan** [comment]
can you maybe download it directly?
```php
$phpWord->save($document->id, 'Word2007', true);
```

**Comment by shihabudheenf** [comment]
I want to save to storage and download later, actually storing and downloading is working fine, but when I am using image in while creating word document, when I am trying to download the same create file later, file is downloading but showing `.zip` extension in browser and google doc preview not supporting. but once downloaded and open in ms word from my system, it is working fine.

**Comment by Fleshen** [comment]
I have almost the same issue but it does not seem to be related to the addImage method.

The mime type of the file will be application/zip regardles of what headers i use when passing the file to the browser.

The main issue is that the preview or the content is not formated when previewing in osx or in the preview or when opening the file in google drive.

If I open the file in Microsoft Word for Mac (v. 15.33) the file looks as it should. If I save the file and remove the compatibility mode then the file will looks better in the osx preview and correct in the google drive preview and when its opened.

I have unziped the file before and after saving it and there are a lot of differences in the xml-files. With mention of 2010 and 2012.

One example of a difference is that the tabs in the exported file are stored as actual tabs while in the updated file they are extracted into tags like this `<w:tab/>`

@troosan Do you have any more insight into this or is it simply that the settings for PhpWord are outdated?

**Comment by troosan** [comment]
@shihabudheenf make sure you send the correct headers
```php
header('Content-Type: application/vnd.openxmlformats-officedocument.wordprocessingml.document');
```

**Comment by troosan** [comment]
@Fleshen indeed, depending on the Word version you are using, different tags might be used for the same feature. Another such difference is in the way pictures are stored. PHPWord is using the old way (w:picture), most modern Word editors will use the new (w:drawing)

**Comment by shihabudheenf** [comment]
@troosan  @Fleshen thank you, now it is working in fine while downloading and in google doc preview but without displaying image :(  (only text content displays)

Everything working good in local computer and ms word. (after downloading)

**Comment by jbelien** [comment]
Same issue here: `Word2007` file generated by `save()` function has `application/zip` as a mime type (instead of correct `application/vnd.openxmlformats-officedocument.wordprocessingml.document`). 😕

**Comment by mishug** [comment]
Same issue

**Comment by mishug** [comment]
Still, images are not visible in google docs.

**Comment by shihabudheenf** [comment]
I used python script to convert word to pdf for preview.
